app/moves/middleware.js

```diff
@@ -39,17 +39,18 @@ const moveTypeNavigationConfig = [
 const movesMiddleware = {
   redirectBaseUrl: (req, res) => {
     const today = format(new Date(), dateFormat)
-    const currentLocation = get(req.session, 'currentLocation.id')
+    const currentLocation = get(req.session, 'currentLocation')
 
     if (currentLocation) {
       const userPermissions = get(req.session, 'user.permissions')
       const canViewProposedMoves = permissions.check(
         'moves:view:proposed',
         userPermissions
       )
-      const url = canViewProposedMoves
-        ? `${req.baseUrl}/week/${today}/${currentLocation}/`
-        : `${req.baseUrl}/day/${today}/${currentLocation}/outgoing`
+      const url =
+        canViewProposedMoves && currentLocation.location_type === 'prison'
```

Comment:
Could we add a unit test to cover all scenarios.

It looks like we're only covering both, or none. What about either just `canViewProposedMoves` or just `currentLocation.location_type === 'prison'`?